app/containers/CustomIcon/index.tsx

```diff
@@ -16,8 +16,8 @@ export interface ICustomIcon extends TextProps {
 	color: string;
 }
 
-const CustomIcon = ({ name, size, color, ...props }: ICustomIcon) => (
+const CustomIcon = ({ name, size, color, style, ...props }: ICustomIcon) => (
 	// @ts-ignore TODO remove this after update @types/react-native to 0.65.0
-	<IconSet name={name} size={size} color={color} {...props} />
+	<IconSet name={name} size={size} color={color} style={StyleSheet.compose({ lineHeight: size }, style)} {...props} />
```

Comment:
```suggestion
	<IconSet name={name} size={size} color={color} style={[{ lineHeight: size }, style]} {...props} />
```
The way the `style` is, if it `style` has `lineHeight`, it will override `lineHeight`, so you don't need this prop.
Now if the `lineHeight` has to overlap the `style`, it should be on the other side, like that: `StyleSheet.compose(style, { lineHeight: size })`